packages/@cdktf/provider-generator/lib/get/generator/models/resource-model.ts

```diff
@@ -178,11 +178,18 @@ export class ResourceModel {
   }
 
   public get namespacedFilePath(): string {
-    return path.join(
-      this.filePath.split("/").slice(0, -1).join("/"),
-      this.namespace!.name,
-      this.structsFolderName
-    );
+    if (this.namespace) {
+      return path.join(
+        this.filePath.split("/").slice(0, -1).join("/"),
```

Comment:
Nitpick: Moving this into a variable with an explanatory name would be nice for readability